<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
    <title>自动化测试报告</title>
    <meta name="generator" content="HTMLTestRunner 1.0.0.0"/>
    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
    <link href="https://cdn.bootcss.com/bootstrap/4.1.1/css/bootstrap.min.css" rel="stylesheet">
    <script src="https://cdn.bootcss.com/jquery/3.3.1/jquery.min.js"></script>
    <script src="https://cdn.bootcss.com/bootstrap/4.1.1/js/bootstrap.min.js"></script>
    <!-- <link href="css/bootstrap.min.css" rel="stylesheet"> -->
    <!--  <script src="js/jquery-3.3.1.min.js"></script> -->
    <!--  <script src="js/bootstrap.min.js"></script> -->
    
<style type="text/css" media="screen">
body        { font-family: Microsoft YaHei,Tahoma,arial,helvetica,sans-serif;padding: 20px; font-size: 80%; }
table       { font-size: 100%; }

/* -- heading ---------------------------------------------------------------------- */
.heading {
    margin-top: 0ex;
    margin-bottom: 1ex;
}

.heading .description {
    margin-top: 4ex;
    margin-bottom: 6ex;
}

/* -- report ------------------------------------------------------------------------ */
#total_row  { font-weight: bold; }
.passCase   { color: #5cb85c; }
.failCase   { color: #d9534f; font-weight: bold; }
.errorCase  { color: #f0ad4e; font-weight: bold; }
.hiddenRow  { display: none; }
.testcase   { margin-left: 2em; }
</style>

</head>
<body >
<script language="javascript" type="text/javascript">
output_list = Array();

/*level 调整增加只显示通过用例的分类 
0:Summary //all hiddenRow
1:Failed  //pt hiddenRow, ft none
2:Pass    //pt none, ft hiddenRow
3:All     //pt none, ft none
*/
function showCase(level) {
    trs = document.getElementsByTagName("tr");
    for (var i = 0; i < trs.length; i++) {
        tr = trs[i];
        id = tr.id;
        if (id.substr(0,2) == 'ft') {
            if (level == 2 || level == 0 ) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
        if (id.substr(0,2) == 'pt') {
            if (level < 2) {
                tr.className = 'hiddenRow';
            }
            else {
                tr.className = '';
            }
        }
    }

    //加入【详细】切换文字变化 
    detail_class=document.getElementsByClassName('detail');
	//console.log(detail_class.length)
	if (level == 3) {
		for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="收起"
		}
	}
	else{
			for (var i = 0; i < detail_class.length; i++){
			detail_class[i].innerHTML="详细"
		}
	}
}

function showClassDetail(cid, count) {
    var id_list = Array(count);
    var toHide = 1;
    for (var i = 0; i < count; i++) {
        //ID修改 点 为 下划线
        tid0 = 't' + cid.substr(1) + '_' + (i+1);
        tid = 'f' + tid0;
        tr = document.getElementById(tid);
        if (!tr) {
            tid = 'p' + tid0;
            tr = document.getElementById(tid);
        }
        id_list[i] = tid;
        if (tr.className) {
            toHide = 0;
        }
    }
    for (var i = 0; i < count; i++) {
        tid = id_list[i];
        //修改点击无法收起的BUG，加入【详细】切换文字变化 
        if (toHide) {
            document.getElementById(tid).className = 'hiddenRow';
            document.getElementById(cid).innerText = "详细"
        }
        else {
            document.getElementById(tid).className = '';
            document.getElementById(cid).innerText = "收起"
        }
    }
}

function html_escape(s) {
    s = s.replace(/&/g,'&amp;');
    s = s.replace(/</g,'&lt;');
    s = s.replace(/>/g,'&gt;');
    return s;
}
</script>
<div class='heading'>
<h1 style="font-family: Microsoft YaHei">自动化测试报告</h1>
<p class='attribute'><strong>测试人员 : </strong> 何冰</p>
<p class='attribute'><strong>开始时间 : </strong> 2021-05-13 22:26:32</p>
<p class='attribute'><strong>合计耗时 : </strong> 0:00:23.409835</p>
<p class='attribute'><strong>测试结果 : </strong> 共 5，错误 5，通过率= 0.00%</p>

<p class='description'>测试环境：Chrome</p>
</div>



<p id='show_detail_line'>
<a class="btn btn-primary" href='javascript:showCase(0)'>通过率 - 0.00% </a>
<a class="btn btn-danger" href='javascript:showCase(1)'>失败 - 0 </a>
<a class="btn btn-success" href='javascript:showCase(2)'>通过 - 0 </a>
<a class="btn btn-info" href='javascript:showCase(3)'>所有 - 5 </a>
</p>
<table id='result_table' class="table table-condensed table-bordered table-hover">
<colgroup>
<col align='left' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
<col align='right' />
</colgroup>
<tr id='header_row' class="text-center success" style="font-weight: bold;font-size: 14px;">
    <td>用例集/测试用例</td>
    <td>总计</td>
    <td>通过</td>
    <td>失败</td>
    <td>错误</td>
    <td>详细</td>
</tr>

<tr class='errorClass warning'>
    <td>test_login_1.login</td>
    <td class="text-center">1</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">1</td>
    <td class="text-center"><a href="javascript:showClassDetail('c1',1)" class="detail" id='c1'>详细</a></td>
</tr>

<tr id='ft1_1' class='none'>
    <td class='errorCase'><div class='testcase'>test_login</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 
    <button id='btn_ft1_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft1_1'>错误</button>
    <div id='div_ft1_1' class="collapse">  -->

    <!-- 默认展开错误信息 -->
    <button id='btn_ft1_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft1_1'>错误</button>
    <div id='div_ft1_1' class="collapse in">
    <pre>
    
ft1_1: Traceback (most recent call last):
  File "C:\Users\何冰\Desktop\Object\C2 selenium\6 自动化测试框架\test_case\test_login_1.py", line 9, in test_login
    self.driver.get("http://127.0.0.1/index.php?m=user&amp;c=public&amp;a=login")
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome=90.0.4430.212)



    </pre>
    </div>
    </td>
</tr>

<tr class='errorClass warning'>
    <td>test_register_1.test_register_1</td>
    <td class="text-center">4</td>
    <td class="text-center">0</td>
    <td class="text-center">0</td>
    <td class="text-center">4</td>
    <td class="text-center"><a href="javascript:showClassDetail('c2',4)" class="detail" id='c2'>详细</a></td>
</tr>

<tr id='ft2_1' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_1___htest1____123456____123456____13252026160____1_123_com__</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 
    <button id='btn_ft2_1' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_1'>错误</button>
    <div id='div_ft2_1' class="collapse">  -->

    <!-- 默认展开错误信息 -->
    <button id='btn_ft2_1' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_1'>错误</button>
    <div id='div_ft2_1' class="collapse in">
    <pre>
    
ft2_1: Traceback (most recent call last):
  File "D:\Python39\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\何冰\Desktop\Object\C2 selenium\6 自动化测试框架\test_case\test_register_1.py", line 15, in test_register
    self.driver.get("http://127.0.0.1/index.php?m=user&amp;c=public&amp;a=reg")
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome=90.0.4430.212)



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_2' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_2___htest2____123456____123456____13252026161____2_123_com__</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 
    <button id='btn_ft2_2' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_2'>错误</button>
    <div id='div_ft2_2' class="collapse">  -->

    <!-- 默认展开错误信息 -->
    <button id='btn_ft2_2' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_2'>错误</button>
    <div id='div_ft2_2' class="collapse in">
    <pre>
    
ft2_2: Traceback (most recent call last):
  File "D:\Python39\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\何冰\Desktop\Object\C2 selenium\6 自动化测试框架\test_case\test_register_1.py", line 15, in test_register
    self.driver.get("http://127.0.0.1/index.php?m=user&amp;c=public&amp;a=reg")
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome=90.0.4430.212)



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_3' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_3___htest3____123456____123456____13252026162____3_123_com__</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 
    <button id='btn_ft2_3' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_3'>错误</button>
    <div id='div_ft2_3' class="collapse">  -->

    <!-- 默认展开错误信息 -->
    <button id='btn_ft2_3' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_3'>错误</button>
    <div id='div_ft2_3' class="collapse in">
    <pre>
    
ft2_3: Traceback (most recent call last):
  File "D:\Python39\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\何冰\Desktop\Object\C2 selenium\6 自动化测试框架\test_case\test_register_1.py", line 15, in test_register
    self.driver.get("http://127.0.0.1/index.php?m=user&amp;c=public&amp;a=reg")
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome=90.0.4430.212)



    </pre>
    </div>
    </td>
</tr>

<tr id='ft2_4' class='none'>
    <td class='errorCase'><div class='testcase'>test_register_4___htest4____123456____123456____13252026163____4_123_com__</div></td>
    <td colspan='5' align='center'>
    <!--默认收起错误信息 
    <button id='btn_ft2_4' type="button"  class="btn btn-danger btn-xs collapsed" data-toggle="collapse" data-target='#div_ft2_4'>错误</button>
    <div id='div_ft2_4' class="collapse">  -->

    <!-- 默认展开错误信息 -->
    <button id='btn_ft2_4' type="button"  class="btn btn-danger btn-xs" data-toggle="collapse" data-target='#div_ft2_4'>错误</button>
    <div id='div_ft2_4' class="collapse in">
    <pre>
    
ft2_4: Traceback (most recent call last):
  File "D:\Python39\lib\site-packages\ddt.py", line 182, in wrapper
    return func(self, *args, **kwargs)
  File "C:\Users\何冰\Desktop\Object\C2 selenium\6 自动化测试框架\test_case\test_register_1.py", line 15, in test_register
    self.driver.get("http://127.0.0.1/index.php?m=user&amp;c=public&amp;a=reg")
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 333, in get
    self.execute(Command.GET, {'url': url})
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\webdriver.py", line 321, in execute
    self.error_handler.check_response(response)
  File "D:\Python39\lib\site-packages\selenium\webdriver\remote\errorhandler.py", line 242, in check_response
    raise exception_class(message, screen, stacktrace)
selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED
  (Session info: chrome=90.0.4430.212)



    </pre>
    </div>
    </td>
</tr>

<tr id='total_row' class="text-center active">
    <td>总计</td>
    <td>5</td>
    <td>0</td>
    <td>0</td>
    <td>5</td>
    <td>通过率：0.00%</td>
</tr>
</table>

<div id='ending'>&nbsp;</div>
    <div style=" position:fixed;right:50px; bottom:30px; width:20px; height:20px;cursor:pointer">
    <a href="#"><span class="glyphicon glyphicon-eject" style = "font-size:30px;" aria-hidden="true">
    </span></a></div>
    

</body>
</html>
